components/HelplineCard.jsx

```diff
@@ -21,6 +19,27 @@ const HelplineCard = ({
 }) => {
     return (
         <div className="w-full bg-white rounded-lg shadow dark:bg-gray-1200 dark:text-gray-300">
+            <div className="w-full">
```

Comment:
Compare this :

https://life-5xhbeu4bk-coronasafe.vercel.app/delhi_(nct)/central_delhi/helpline

with

https://life.coronasafe.network/delhi_(nct)/central_delhi/helpline

<img width="897" alt="Screenshot 2021-04-24 at 11 22 24 PM" src="https://user-images.githubusercontent.com/61158210/115968144-efdc6880-a553-11eb-8855-788c9af9638e.png">

<img width="884" alt="Screenshot 2021-04-24 at 11 22 11 PM" src="https://user-images.githubusercontent.com/61158210/115968141-e81cc400-a553-11eb-9c80-07545255303a.png">

Card's Layout break

Reply:
The code that's in the main is the same as what's in mine that affects that layout.

Reply:
The cards have become unmanageable.

Reply:
Fixed 📍